src/Hedgehog/Report.fs

```diff
@@ -9,10 +9,16 @@ type RecheckType =
     | None
     | CSharp
     | FSharp
-
-type FailureData = {
+    
+    
+[<Struct>]
+type RecheckData = internal {
```

Comment:
I'm not a big fan of the name RecheckData, from a consumer POV. Perhaps something like RecheckSeed? This doesn't need to be the name of the internal type... maybe just on the public API? e.g.

```fsharp
let recheck (recheckSeed: string) (p : Property<unit>) : unit =
```

Could perhaps make it even simpler:

```fsharp
let recheck (seed: string) (p : Property<unit>) : unit =
```

Reply:
I like `recheckSeed`.  I think `seed` is too confusing.

Reply:
But I think `recheckData` is better, even for the user.